<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertIncome</class>
 <widget class="QWidget" name="insertIncome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>数额</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="countInput"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>备注</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="remarksInput"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="OkButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
